### PR TITLE
Add unit tests for UnifiedTransformer

### PR DESCRIPTION
**Pull request by FrostML**
### PR types

Others

### PR changes

Others

### Description

Add unit tests for UnifiedTransformer.

### CODE REVIEW

File: paddlenlp/transformers/unified_transformer/modeling.py [hunk 1]
```diff
@@ -384,6 +422,10 @@ def forward(self,
                     is_split_into_words=False)
                 outputs = model(**inputs)
         """
+        if attention_mask is None:
+            attention_mask = ((input_ids == self.pad_token_id).astype(
+                paddle.get_default_dtype()) * -1e9).unsqueeze([1, 2])
```
Comment (guoshengCS) [inline paddlenlp/transformers/unified_transformer/modeling.py]: 现在统一用-1e4了，之前改过一波

File: paddlenlp/transformers/unified_transformer/modeling.py [hunk 2]
```diff
@@ -558,23 +600,55 @@ def adjust_logits_during_generation(self, logits):
...
+                    num_pad).astype("int64")
+            position_ids.stop_gradient = True
+
+        if token_type_ids is None:
+            token_type_ids = paddle.zeros_like(input_ids, dtype="int64")
+            token_type_ids.stop_gradient = True
+
+        if attention_mask is None:
+            attention_mask = ((input_ids == self.pad_token_id).astype(
+                paddle.get_default_dtype()) * -1e9).unsqueeze([1, 2])
```
Comment (guoshengCS) [inline paddlenlp/transformers/unified_transformer/modeling.py]: 同上，现在统一用-1e4了，之前改过一波

File: paddlenlp/transformers/unified_transformer/modeling.py (hunk 1 above)
Comment (FrostML) [reply paddlenlp/transformers/unified_transformer/modeling.py]: Done.

File: paddlenlp/transformers/unified_transformer/modeling.py (hunk 2 above)
Comment (FrostML) [reply paddlenlp/transformers/unified_transformer/modeling.py]: Done.

**Review by guoshengCS — APPROVED** [review]

**Pull request merged**